examples/fastspeech/train_fastspeech.py

```diff
@@ -324,6 +331,10 @@ def main():
         )
         fastspeech._build()
         fastspeech.summary()
+        
+        if len(args.pretrained) > 1:
+            print("Loading pretrained parameters...")
```

Comment:
can you replace this by:

`logging.info(f"Successfully loaded pretrained weight from {args.pretrained}.")`

after the line `fastspeech.load_weights(...)`. Do the same for other files :D.